.circleci/config.yml

```
@@ -62,6 +62,18 @@ commands:
       - store_artifacts:
           path: ~/artifacts
 
+  makesmoke:
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ./
+      - run:
+          name: Placeholder - Spin up example in Docker
```

This is great, we should do it!

```suggestion
          name: Spin up example in Docker
```